tests/acceptance/features/bootstrap/SearchContext.php

```diff
@@ -133,36 +133,28 @@ public function fileOrFolderInTheSearchResultShouldContainProperties(
 		$user = $this->featureContext->getActualUsername($user);
 		$this->featureContext->verifyTableNodeColumns($properties, ['name', 'value']);
 		$properties = $properties->getHash();
-		$fileResult = $this->featureContext->findEntryFromPropfindResponse(
-			$path,
-			$user,
-			"REPORT",
+		$fileResult = $this->featureContext->findEntryFromSearchResponse(
+			$path
 		);
+
 		Assert::assertNotFalse(
 			$fileResult,
 			"could not find file/folder '$path'"
 		);
-		$fileProperties = $fileResult['value'][1]['value'][0]['value'];
 		foreach ($properties as $property) {
-			$foundProperty = false;
 			$property['value'] = $this->featureContext->substituteInLineCodes(
 				$property['value'],
 				$user
 			);
-			foreach ($fileProperties as $fileProperty) {
-				if ($fileProperty['name'] === $property['name']) {
-					Assert::assertMatchesRegularExpression(
-						"/" . $property['value'] . "/",
-						$fileProperty['value']
-					);
-					$foundProperty = true;
-					break;
-				}
+			$fileResultProperty = $fileResult->xpath("d:propstat//" . $property['name']);
+			if ($fileResultProperty) {
+				Assert::assertMatchesRegularExpression(
+					"/" . $property['value'] . "/",
+					\trim((string)$fileResultProperty[0])
+				);
+			} else {
+				throw new Error("Could not find property '" . $property['name'] . "'");
 			}
```

Comment:
```suggestion
				return Assert::assertMatchesRegularExpression(
					"/" . $property['value'] . "/",
					\trim((string)$fileResultProperty[0])
				);
			}
			throw new Error("Could not find property '" . $property['name'] . "'");
```

Reply:
It is a step definition i think it is not to do a return

Reply:
returning should not matter. It's just to break the code